### DISCUSSION TITLE
Possible unneeded setup_requires for "setuptools_scm_git_archive"

### DISCUSSION THREAD

**Discussion by fdebrabander** [pycontribs/jira#1365] [Q&A] [answered]
In the setup.cfg there is a dependency on `setuptools_scm_git_archive`.

```
setup_requires =
    setuptools_scm >= 1.15.0
    setuptools_scm_git_archive >= 1.0
```

The documentation over at https://github.com/Changaco/setuptools_scm_git_archive states that there are two files needed for this to work `.git_archival.txt` and `.gitattributes`. Both files are missing in this repository.

Is this indeed a dependency that can be removed or am I missing something?

**Comment by adehad** [comment, marked as answer]
Good spot, probably can be removed, was added around 3 years ago from the looks of it so might have been a workaround at the time
